catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/SubmitStandingQueryRequestImpl.java

```diff
@@ -640,6 +569,91 @@ protected DAGQueryResult getData(long queryTime) {
       return result;
     }
 
+    private QueryRequestImpl getQueryRequest(Filter parsedFilter) {
+      QueryRequestImpl catalogQueryRequest;
+      if (querySources == null || querySources.isEmpty()) {
+        LOGGER.trace("Query request will be local, no sources specified: {}", parsedFilter);
+        catalogQueryRequest = new QueryRequestImpl(catalogQuery);
+      } else {
+        if (LOGGER.isTraceEnabled()) {
+          String sourceList = querySources.stream().sorted().collect(Collectors.joining(", "));
+          LOGGER.trace("Query will use the following sources: {}", sourceList);
+        }
+        catalogQueryRequest = new QueryRequestImpl(catalogQuery, false, querySources, null);
+      }
+      return catalogQueryRequest;
+    }
+
+    private Filter getFilter(long queryTime) {
+      Filter parsedFilter = bqsFilter;
+      if (!moreResultsAvailOnLastQuery && queryTime > 0) {
+        parsedFilter =
+            filterBuilder.allOf(
+                bqsFilter,
+                filterBuilder.attribute(Metacard.MODIFIED).is().after().date(new Date(queryTime)));
```

Comment:
Unfortunately, `Core` doesn't provide the `DEFAULT_TAG` "resource" , so I can't replace all of them. Since I can't replace them all, I don't know if we should replace the others... Who else would want to weigh in on this? Maybe @brendan-hofmann ?